libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFSDKHybridLogger.h

```diff
@@ -0,0 +1,88 @@
+/*
+ SFSDKHybridLogger.h
+ SalesforceHybridSDK
+ 
+ Created by Bharath Hariharan on 6/22/17.
+ 
+ Copyright (c) 2017-present, salesforce.com, inc. All rights reserved.
+ 
+ Redistribution and use of this software in source and binary forms, with or without modification,
+ are permitted provided that the following conditions are met:
+ * Redistributions of source code must retain the above copyright notice, this list of conditions
+ and the following disclaimer.
+ * Redistributions in binary form must reproduce the above copyright notice, this list of
+ conditions and the following disclaimer in the documentation and/or other materials provided
+ with the distribution.
+ * Neither the name of salesforce.com, inc. nor the names of its contributors may be used to
+ endorse or promote products derived from this software without specific prior written
+ permission of salesforce.com, inc.
+ 
+ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR
+ IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+ CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+ WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY
+ WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#import <SalesforceAnalytics/SFSDKLogger.h>
+
+@interface SFSDKHybridLogger : NSObject
+
+/**
+ * Returns current log level used by this logger.
+ *
+ * @return Current log level.
+ */
++ (DDLogLevel)curLogLevel;
+
+/**
+ * Sets log level to be used by this logger.
+ *
+ * @param logLevel Log level.
+ */
++ (void)setLogLevel:(DDLogLevel)logLevel;
+
+/**
+ * Logs an error log line.
+ *
+ * @param class Class.
+ * @param message Log message.
+ */
++ (void)e:(nonnull Class)class message:(nonnull NSString *)message;
```

Comment:
Should we make the e,w,i,v,d methods [variadic functions](https://stackoverflow.com/questions/4804674/how-to-create-variable-argument-methods-in-objective-c), to safe the callers the ned to do ```[NSString stringWithFormat:...]```